### PR TITLE
Fixing errors suggested by codacy.

### PR DESCRIPTION
**Pull request by kegor**

### CODE REVIEW

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 31
- Added 2
           

Clones added
============
- src/Products/Common/Util/Comparator/FileTypeComparator.cs  1
- src/Products/Common/Util/Comparator/FileNameComparator.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5038425&bid=16594409)

File: src/Products/Comparison/Service/ComparisonServiceImpl.cs [hunk 1]
```diff
@@ -175,10 +170,9 @@ public PageDescriptionEntity LoadDocumentPage(PostedDataEntity postedData)
                 loadedPage.width = resultImages[pageNumber - 1].Width;
 
             }
-            catch (System.Exception ex)
+            catch (Exception)
```
Comment (egorovpavel) [inline src/Products/Comparison/Service/ComparisonServiceImpl.cs]: Codacy Issue found: [Add logic to this catch clause or eliminate it and rethrow the exception automatically.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5038425)

File: src/Products/Comparison/Service/ComparisonServiceImpl.cs [hunk 2]
```diff
@@ -243,7 +237,7 @@ private ICompareResult CompareFiles(CompareRequest compareRequest)
 
             if (compareResult == null)
             {
-                throw new Exception("Something went wrong. We've got null result.");
+                throw new NullReferenceException("Something went wrong. We've got null result.");
```
Comment (egorovpavel) [inline src/Products/Comparison/Service/ComparisonServiceImpl.cs]: Codacy Issue found: ['System.NullReferenceException' should not be thrown by user code.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5038425)

**Comment by egorovpavel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 32
           

Clones added
============
- src/Products/Common/Util/Comparator/FileTypeComparator.cs  1
- src/Products/Common/Util/Comparator/FileNameComparator.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/GroupDocs/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5038425&bid=16594409)

**Comment by bobkovalex** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 33
           

Clones added
============
- src/Products/Common/Util/Comparator/FileTypeComparator.cs  1
- src/Products/Common/Util/Comparator/FileNameComparator.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/groupdocs-comparison/GroupDocs.Comparison-for-.NET-MVC/pullRequest?prid=5038426&bid=16594410)

**Pull request merged**